templates/config/nginx/common/redis.conf

```diff
@@ -0,0 +1,57 @@
+# Redis NGINX CONFIGURATION
+# DO NOT MODIFY, ALL CHANGES LOST AFTER UPDATE EasyEngine (ee)
+set $skip_cache 0;
+# POST requests and URL with a query string should always go to php
+if ($request_method = POST) {
+  set $skip_cache 1;
+}
+if ($query_string != "") {
+  set $skip_cache 1;
+}
+# Don't cache URL containing the following segments
+if ($request_uri ~* "(/wp-admin/|/xmlrpc.php|wp-.*.php|index.php|/feed/|sitemap(_index)?.xml|[a-z0-9_-]+-sitemap([0-9]+)?.xml)") {
+  set $skip_cache 1;
+}
+# Don't use the cache for logged in users or recent commenter or customer with items in cart
+if ($http_cookie ~* "comment_author|wordpress_[a-f0-9]+|wp-postpass|wordpress_no_cache|wordpress_logged_in|woocommerce_items_in_cart") {
+  set $skip_cache 1;
+}
+# Use cached or actual file if they exists, Otherwise pass request to WordPress
+location / {
+  try_files $uri $uri/ /index.php?$args;
+}
+
+location /redis-fetch {
+    internal  ;
+    set  $redis_key $args;
+    redis_pass  ee4_redis:6379;
+}
+location /redis-store {
+    internal  ;
+    set_unescape_uri $key $arg_key ;
+    redis2_query  set $key $echo_request_body;
+    redis2_query expire $key 14400;
+    redis2_pass  ee4_redis:6379;
```

Comment:
Can we stop using the word `ee4`? There will be only one `ee` after this.

Reply:
Yes. Noted.